src/lib/containers/entity/metadata/EntityModal.tsx

```diff
@@ -112,82 +170,41 @@ export const EntityModal: React.FC<EntityModalProps> = ({
               })}
             </div>
           ) : null}
-          <>
-            {currentTab === EntityModalTabs.ANNOTATIONS && (
-              <>
-                {isInEditMode ? (
-                  <SynapseErrorBoundary>
-                    <SchemaDrivenAnnotationEditor
-                      entityId={entityId}
-                      onSuccess={() => {
-                        setIsInEditMode(false)
-                      }}
-                      onCancel={() => setIsInEditMode(false)}
-                    />
-                  </SynapseErrorBoundary>
-                ) : (
-                  <AnnotationsTable
-                    entityId={entityId}
-                    versionNumber={versionNumber}
-                  />
-                )}
-              </>
-            )}
-            {currentTab === EntityModalTabs.METADATA && (
-              <MetadataTable
+          <div
+            style={
+              currentTab === EntityModalTabs.ANNOTATIONS
+                ? {}
+                : { display: 'none' }
+            }
+          >
```

Comment:
Use `display: none` instead of conditional rendering to prevent a flicker that happens between the old tab content unloading and the new tab content loading.

Aside: I wonder if any of React 18's new features prevent this problem